catalog/transformer/catalog-transformer-tika-input/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -13,14 +13,36 @@
  **/
  -->
 <blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
-        <!-- The tika input transformer programmatically registers itself as a service -->
+    <!-- The tika input transformer programmatically registers itself as a service -->
     <bean id="tikaTransformer" class="ddf.catalog.transformer.input.tika.TikaInputTransformer">
-		    <argument ref="blueprintBundleContext"/>
-		</bean>
+        <argument ref="blueprintBundleContext"/>
+        <argument ref="genericMetacardType"/>
+    </bean>
 
-    <reference-list id="contentExtractors" interface="ddf.catalog.content.operation.ContentMetadataExtractor"
+    <bean id="genericMetacardType" class="ddf.catalog.data.impl.MetacardTypeImpl">
```

Comment:
I'm not crazy about calling this `generic`.  I guess it's a catch all, just wondering if we can give it a more specific name and document when someone would want to use this type versus another type.